src/components/calcite-radio-group/calcite-radio-group.tsx

```diff
@@ -112,17 +111,12 @@ export class CalciteRadioGroup implements LabelableComponent, FormComponent {
   //
   //--------------------------------------------------------------------------
 
-  connectedCallback(): void {
-    const items = this.getItems();
-    const lastChecked = Array.from(items)
-      .filter((item) => item.checked)
-      .pop();
+  componentWillLoad(): void {
+    this.selectInitialItem();
+  }
```

Comment:
Is this part of the behavior spec for this component?  This would mean that there is no way to make this component optional in a form, and it diverts from native radio behavior.

Reply:
The name is a bit misleading, it will only select an item if it has the checked property. Otherwise, it makes the first option tabbable.